app/Http/Controllers/V2/Polygons/ViewAllSitesPolygonsForProjectController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers\V2\Polygons;
+
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\SitePolygon\SitePolygonResource;
+use App\Models\V2\Projects\Project;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\ResourceCollection;
+
+class ViewAllSitesPolygonsForProjectController extends Controller
+{
+    public function __invoke(Request $request, Project $project): ResourceCollection
+    {
+        $sitePolygons = $project->with('sitePolygons')->get()->pluck('sitePolygons')->flatten();
```

Comment:
I'm not sure what the `with()` or `flatten()` are accomplishing here. Does `$project->sitePolygons()->get()` not give you the correct set?

Reply:
yes, it is simpler this way.
changed.